### DISCUSSION TITLE
Add an option to allow the bottom dock to be full width under the left dock

### DISCUSSION THREAD

**Discussion by metcalfc** [zed-industries/zed#23916] [Feature Requests]
Summary: Sometimes you want more space in the terminal, add a `justified` alignment option for the terminal panel.

Sometimes you want more space in the terminal. You can close the left dock. But I often still want to see the tree. I'd like an option to expand the terminal (bottom dock) under the left dock. VS Code calls this `justified` alignment.

https://twitter.com/i/status/1776271247925498277

**Comment by mikayla-maki** [comment]
Thanks for this feature request! We have a PR up for this feature:

https://github.com/zed-industries/zed/pull/21337
